### ISSUE TITLE
Accept / reject appointment via link in mail

### ISSUE THREAD

**Issue by tlwt** [Admidio/admidio#237]
It would be great if an appointment could be accepted link provided in the email. Use case - we have an informal team playing every Friday and we easily would like to know if we are enough / who participates.

**Comment by Fasse** [comment]
This is a good idea but before we can do something here we have a lot of other feature request within this module.

**Comment by thomas-rcv** [comment]
As I know there is already an issue for this feature.
It is descripted as a generally notification system for participants.

**Comment by Fasse** [comment]
Duplicate of #477 
Because of the detailed description in the other issue, I will mark this one as duplicate.
